test/setup_contracts.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+docker-compose up -d ganache-cli
+cd dex-contracts
+npm install --production
+npx wait-port -t 30000 8545
+npx truffle migrate
```

Comment:
minor: You could use the migration flag --onlyMigrateStableX or --onlyMigrateDfusion to save some milliseconds

Reply:
This would affect the contract addresses though. Not sure if it's worth it.